### ISSUE TITLE
How to add Connector from top to bottom

### ISSUE THREAD

**Issue by piyushSinghalDemo** [sdrdis/jquery.flowchart#87]
Currently we have connecter from left to right but is it possible to add connector from top to bottom, Actually I want to show flow of diagram, So it was required

**Comment by runette** [comment]
Seems to me that  is a huge change to flowchart. Not likely to happen
unless you do it yourself.

I use OrgChart for doing this , although without the dynamic editing
function :

https://github.com/dabeng/OrgChart

**Comment by sdrdis** [comment]
It is indeed not a functionality yet. Since you are not the first one to ask this, I mark this as a possible enhancement.

**Comment by piyushSinghalDemo** [comment]
@runette yes actually i also find alternative of this library, I started to work with Jointjs to achive all this functionality.That is also providing community version with good documentation.
"https://resources.jointjs.com/tutorial"

**Comment by runette** [comment]
Interesting. Thanks
